website/content/api-docs/secret/pki/issuance.mdx

```diff
@@ -790,12 +796,10 @@ $ curl \
         "cert_role": "cert1"
       },
     "default_path_policy": "sign-verbatim",
+    "disabled_validations": ["DisableMatchingKeyIdValidation"],
     "enabled": true,
     "enable_sentinel_parsing": true,
     "last_updated": "2024-02-02T10:49:20-05:00"
   }
 }
 ```
```

Comment:
Why are we deleting end-quotes?

Reply:
That's a mistake, I'll fix it.

Reply:
Actually, the end quotes are there, this is just funky coloring of the diff.